Simulation: more access to camera parameters

damienjadeduff opened the pull request:
This gives the user of the simulation module the ability to retrieve camera parameters and also to set the clipping planes, which earlier were set to arbitrary values.
We are using the simulation module in its current state and we expect others are too. 
Kind regards.

stefanbuettner commented on simulation/include/pcl/simulation/range_likelihood.h:
```diff
@@ -145,6 +164,18 @@ namespace pcl
         getScoreBuffer ();
 
+        float 
+        getZNear(){ return z_near_; }
```
Space before opening parenthesis.

stefanbuettner commented on simulation/include/pcl/simulation/range_likelihood.h:
```diff
@@ -99,6 +99,25 @@ namespace pcl
+                                      float &camera_fx_in,
+                                      float &camera_fy_in,
+                                      float &camera_cx_in,
+                                      float &camera_cy_in)
```
Indentation: Missing space.

stefanbuettner commented:
Hey damienjadeduff,

firstly, I think that's a good approach you're following there :+1:
Looks good to me, if you fix some minor coding style issues.

Cheers,
Stefan

SergioRAgostinho commented:
Please squash the commits into a single one.

damienjadeduff commented:
I made the requested white-space changes and squashed the commits. I made an assumption about the 2nd request and changed the pre-existing set() function in the same way. Regards.

stefanbuettner commented:
Looks good to me
